Assets/_about-main-section-info.scss

```diff
@@ -0,0 +1,100 @@
+main {
+  border-bottom: 1px solid $light-color;
+}
+
+.main-section-info {
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+  color: $dark-color;
+  padding: 2rem 0;
+  font-family: $secondary-font;
+}
```

Comment:
- [ ] As per project requirements, pages use the exact same colours, for text, backgrounds, borders, etc, from the original design. Please try to follow the colour, text as per design.
- [ ] Basic color swatch: #272A31, #EC5242, #D3D3D3

Reply:
Actually, Emily from Student success posted a theme saying we can personalize the colors, no need to use the same color.

Reply:
- Okay, If that is the case then you can go ahead with your personalized colour choice. Just fix the other required changes. 

Happy coding. 🚀

Reply:
Yes it was on slack cohort 1 threat but it got automatically erased because was days ago and I'm not paying using of slack app.